### DISCUSSION TITLE
how to filter ip addresses indicator ONLY?

### DISCUSSION THREAD

**Discussion by NSH531** [CrowdStrike/falconpy#628] [Q&A] [answered]
thanks,netanel

**Comment by jshcodes** [comment, marked as answer]
Hi @NSH531 -

You should be able to use the `filter` keyword to limit your results to just IP addresses.  (A complete list of available filters can be found here: https://falconpy.io/Service-Collections/Intel.html#queryintelindicatorentities)

```python
result = intel.query_indicator_entities(filter="type:'ip_address'")

# OR

result = intel.query_indicator_entities(filter="type:'ip_address_block'")
``` 

Uber class would do the same thing with slightly different syntax:

```python
result = falcon.command("QueryIntelIndicatorEntities", filter="type:'ip_address'")
```
